server/tests/unit/common/test_api.py

```diff
@@ -843,6 +841,43 @@ def test_dataset_metadata_api_fails_gracefully_on_connection_failure(self, mock_
 
         self.assertEqual(result.status_code, HTTPStatus.BAD_REQUEST)
 
+class TestConfigEndpoint(BaseTest):
```

Comment:
Note that the tests in the other suites aren't independent, so adding this to an existing suite doesn't work (it causes other tests to fail). Rewriting this file to have independent tests would be great.

Reply:
Are you referring to the server config setup? or something else that prevents the test from being independent?